app/controllers/microposts_controller.rb

```diff
@@ -43,7 +43,8 @@ def create
   # PATCH/PUT /microposts/1 or /microposts/1.json
   def update
     @micropost = Micropost.find(params[:id])
-    @micropost.update_columns(content: micropost_params[:content])
+    @micropost.update_columns(content: micropost_params[:content], published_on: \
+                              micropost_params[:published_on])
```

Comment:
@HrisikeshSinghaChowdhury please keep key, value in same line

Reply:
ok